src/rabbit_mgmt_wm_vhost.erl

```diff
@@ -65,17 +65,25 @@ to_json(ReqData, Context) ->
 
 accept_content(ReqData0, Context = #context{user = #user{username = Username}}) ->
     Name = id(ReqData0),
+    {ok, Body, ReqData1} = rabbit_mgmt_util:read_complete_body(ReqData0),
```

Comment:
`rabbit_mgmt_util:with_decode/4` already does this. This function seems to do the same thing twice now.

Reply:
Corrected.